code/obj/blob.dm

```diff
@@ -1067,6 +1077,51 @@
 	update_icon(override_parent)
 		return
 
+/datum/contextAction/blob_tile_upgrade
```

Comment:
It'd be really cool if these showed the cooldowns, maybe just shove the actual ability button in the vis_contents of the context option?